exercises/toRoman/toRoman.js

```diff
@@ -15,14 +15,47 @@
  * @returns {string} The input integer's Roman numeral representation
  */
 function toRoman(num) {
-  // This is your job. :)
+  let ones = ["I","IV","V","VI","VII","VIII","IX","X"];
+  let tens = ["X","L","C"];
+  let hundereds = ["D","M"];
+  let n = num.toString();
+  let ROM = [];
+  ROM = n.split("");
+  console.log(ROM);
+
+  console.log(ROM[ROM.length-1]);
+
+  
+  for(i = 0; i <= n; i++)
+  { 
+
+    while(ROM[ROM.length-1] === '1' || ROM[ROM.length-1] === '2' || ROM[ROM.length-1] === '3')
+      {ROMAN_NUM += ones[0]}
+
+    if(ROM[ROM.length-1] === '4')
```

Comment:
Is this how you'd translate a number into roman numerals by hand? Make sure you're clear on how you'd do it by hand before writing code.